core/embed/io/nfc_storage/st25r3916b/devices/nfc_device.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * This file is part of the Trezor project, https://trezor.io/
+ *
+ * Copyright (c) SatoshiLabs
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <trezor_types.h>
+
+/*******************************************************************************
+ *                                 ST25TV
+ ******************************************************************************/
+
+bool st25tv_identify(void);
+
+bool st25tv_check_connection(void);
+
+bool st25tv_get_mem_struct(nfc_storage_mem_struct_t *mem_struct);
```

Comment:
we should probably include #include <io/nfc_storage.h> if we are using it in this header